service/src/test/java/bio/terra/pipelines/common/utils/FlightUtilsTest.java

```diff
@@ -225,4 +226,42 @@ void flightComplete_running_isNotComplete() {
     // flightComplete returns True if flight is SUCCESS, ERROR, or FATAL
     assertFalse(FlightUtils.flightComplete(flightState));
   }
+
+  @Test
+  void isContextInvalid() {
+    // null context is invalid
+    assertTrue(FlightUtils.isContextInvalid(null));
+
+    // context with null working map is invalid
+    TestFlightContext context = new TestFlightContext();
+    context.workingMap(null);
+    assertTrue(FlightUtils.isContextInvalid(context));
+
+    // context with working map is not invalid
+    TestFlightContext contextWithWorkingMap = new TestFlightContext();
+    assertFalse(FlightUtils.isContextInvalid(contextWithWorkingMap));
+  }
+
+  @Test
+  void inputParametersContainTrue_true() {
```

Comment:
i am not loving the mix of camel and snake case.  I think you its just as understandable as just `inputParametersContainsTrue` and the one after this as `inputParametersContainsFalse` and `inputParametersContainsNoKey` or just probably better yet just one function that tests all three scenarios since they're not very complicated with a comment or two